packages/kit/src/runtime/client/client.js

```diff
@@ -1017,7 +1008,10 @@ export function create_client(app, target) {
 		url = intent?.url || url;
 
 		// abort if user navigated during update
-		if (token !== nav_token) return false;
+		if (token !== nav_token) {
+			nav.reject(new Error('navigation was aborted'));
```

Comment:
this is getting thrown when updating query params via `goto()`

Reply:
This should be fixed by 1.24.1 (tried to update ?) https://github.com/sveltejs/kit/pull/10666